.travis.yml

```diff
@@ -66,7 +66,7 @@ install:
         pip install virtualenv
     elif [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
         pip install --user virtualenv
-        curl https://cmake.org/files/v3.8/cmake-3.8.0-Linux-x86_64.tar.gz | tar xvz
+        curl https://cmake.org/files/v3.8/cmake-3.8.0-Linux-x86_64.tar.gz | tar xz
```

Comment:
```
curl -Ls https://cmake.org/files/v3.8/cmake-3.8.0-Linux-x86_64.tar.gz | tar -xz -C cmake --strip-components=1
PATH=$PWD/cmake/bin${PATH:+:$PATH}
```
Will silence both cURL _and_ tar and save to a `cmake` folder. The `LD_LIBRARY_PATH` is unnecessary.

A question: since Autocmake is CMake-version agnostic, shouldn't the tests be run using the oldest supported version _aka_ 2.8.12?

Reply:
Yes, we need to talk about versions - I will open a new issue on that. I will fix the other things.

Reply:
Version question is followed-up in #234.